Snippets/ASP/ASTP_3/Behaviors.cs

```diff
@@ -121,14 +126,9 @@ public class Registration : RegisterStep
             base(nameof(OrderIdAsPartitionKeyBehavior),
                 typeof(OrderIdAsPartitionKeyBehavior),
                 "Determines the PartitionKey from the logical message",
-                b => new OrderIdAsPartitionKeyBehavior(b.Build<IProvidePartitionKeyFromSagaId>()))
-        {
+                b => new OrderIdAsPartitionKeyBehavior(b.Build<IProvidePartitionKeyFromSagaId>())) =>
             InsertBefore(nameof(LogicalOutboxBehavior));
-        }
     }
-
```

Comment:
Same comment as before, where do we keep the private members? At the bottom or the top? And why is it different in snippets as opposed to our (framework) code?

Reply:
I'll put them back at the bottom so we can deal with this separately from this PR.